docs/guides/introduction.md

```diff
@@ -22,47 +14,54 @@ ZKPool aims to provide the following capabilities:
 
 Initially, ZKPool will assume the risk and stake on behalf of all the provers connected to the pool. The pool is open for registered provers to connect to.
 
-## General flow
+## General Flow
 
 Currently, the general flow is as follows:
 
 1. Register on [https://zkpool.io](https://zkpool.io/), generate the user account's unique access key.
 2. Use the one command installation script to connect to ZKPool.
 3. Login to [https://zkpool.io](https://zkpool.io/) to view prover machine status and rewards.
 
-## Hardware requirements
+## Releases
 
-The Taiko project supports various provers to join, CPU, GPU, and so on. To avoid dominance by hardware performance, it has a relatively long proof window that CPU devices can join as a prover.
+### ZKPool A1
 
-ZKPool will provide both CPU and GPU binaries (coming soon), and it will also open-source the prover client codes to support 3rd party to build its hardware binary.
+#### Taiko Project Support
 
-The hardware that meets the minimum requirements can run CPU prover binary:
+Taiko is a decentralized, Ethereum-equivalent zk-rollup project with Type 1 zkEVM technology.
 
-- 8 or 16 core CPU
-- 32 GB memory
+ZKPool A1 is dedicated to the [Taiko Alpha-5 (A5) Testnet](https://taiko.mirror.xyz/0jc5XA5RVjOUNVdRdGmw9prlsC4deA1XgZJppmXh8rs), and supports users who want to join as individual provers.
+
+#### Hardware requirements
 
-In Taiko A4, the proof time on the above hardware may take more than 10 minutes.
+ZKPool supports various provers to join, CPU, GPU, and so on. 
 
-Note from [Taiko](https://taiko.mirror.xyz/HJCWBluTwmNyWRkhzIXXr0k5xAaalRNtmlyDMJTu_ws): this is not representative of what a full ZK-EVM proof will eventually cost as not all circuits are included yet. More circuits are being integrated as we go.
+ZKPool will provide both CPU and GPU binaries (coming soon), and it also open-source [the prover client codes](https://github.com/aoraki-labs/zkpool-prover) to support 3rd party to build its hardware binary.
+
+The hardware that meets the minimum requirements can run CPU prover binary:
 
-In the future, the hardware requirements may be updated as Taiko covers more circuits.
+- 8 or 16 core CPU
+- 32 GB memory
 
-## Reward distribution
+#### Reward distribution
 
 According to [Taiko](https://taiko.mirror.xyz/HJCWBluTwmNyWRkhzIXXr0k5xAaalRNtmlyDMJTu_ws):
```

Comment:
This is an old link, please check the quoted words are still applied?

Reply:
done.